pandas/core/generic.py

```diff
@@ -648,13 +648,14 @@ def transpose(self, *args, **kwargs):
         copy : boolean, default False
             Make a copy of the underlying data. Mixed-dtype data will
             always result in a copy
+        kwargs : Additional keyword arguments will be passed to the function
```

Comment:
There is a standard for args and kwargs so here you just want `**kwargs` on this line (no type require) and move the description down one line